server/lib/compareUserPassword.ts

```diff
@@ -15,8 +18,8 @@ export function compareUserPassword(user, pass) {
 	}
 
 	const password = pass.plain || {
-		digest: pass.sha256.toLowerCase(),
-		algorithm: 'sha-256',
+		digest: pass.sha256 ? pass.sha256.toLowerCase() : '',
```

Comment:
Why not `pass?.sha256?.toLowerCase() || ''`?

Reply:
Oh, good suggestion. I'll change this line 🙂

Reply:
This gave me an error in my editor, but nothing on lint (as it should). So I think that's ok 😆